SETUP/manual_web/split_test/h_split.js

```diff
@@ -0,0 +1,12 @@
+/* global $ splitControl */
```

Comment:
Consider renaming the test-centric JS files so they match the name of the PHP files they are used it so it's clear what goes with what at a glance.

Reply:
done this.